neqo-transport/src/connection/tests/close.rs

```diff
@@ -57,7 +64,14 @@ fn connection_close_with_long_reason_string() {
     let long_reason = String::from_utf8([0x61; 2048].to_vec()).unwrap();
     client.close(now, 42, long_reason);
 
+    let stats_before = client.stats().frame_tx;
     let out = client.process(None, now);
+    let stats_after = client.stats().frame_tx;
+    assert_eq!(
+        stats_after.connection_close,
+        stats_before.connection_close + 1
+    );
+    assert_eq!(stats_after.ack, stats_before.ack + 1);
```

Comment:
Why are you seeing an ACK sent in this case?  I thought that the close would squeeze out the ACK.

Reply:
The reason phrase is limited to 256 in https://github.com/mozilla/neqo/blob/52675c358c29a3a1c434050b93719cf88c86c61a/neqo-transport/src/connection/state.rs#L176-L183, which leaves enough space for the ACK.

Reply:
Ah.  I forgot about that.  That means that we're not really testing the case where the close squeezes the ACK out.  Should we?  (Looking at the code, I don't think that we can.  I don't think we ever have more than two spaces to send a close in, and at 256 bytes, I doubt we'd hit the datagram size limit.)

Reply:
I don't think so either.